backend/routes/user.go

```diff
@@ -89,3 +94,30 @@ func getPixelCount(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write([]byte(fmt.Sprintf(`{"count": %d}`, count)))
 }
+
+func getLastPlacedPixel(w http.ResponseWriter, r *http.Request) {
+	address := r.URL.Query().Get("address")
+	var lastPlacedTime time.Time
+	err := core.ArtPeaceBackend.Databases.Postgres.QueryRow(context.Background(), "SELECT time FROM LastPlacedTime WHERE address = $1", address).Scan(&lastPlacedTime)
+	if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Please add a check like this here https://github.com/fishonamos/art-peace/blob/1f7b4c4c8519a5d1d4170130b3beb0b020d3355d/backend/routes/user.go#L31

If there are no rows for the address, treat the last placed time as 0

Reply:
ok